src/build123d/topology.py

```diff
@@ -366,6 +366,7 @@
 
 Shapes = Literal["Vertex", "Edge", "Wire", "Face", "Shell", "Solid", "Compound"]
 
+TrimmingTool = Union[Plane,"Shell", "Face"]
```

Comment:
I think there is a space missing between `Plane, "Shell"` (just don't want this to randomly show up in a future unrelated PR).